### PR TITLE
PJ_SIM_SUNBNB_2023-12 [FE] Home Page Card Component

### PR DESCRIPTION
**Pull request by ThereseRoca**
# Issue Link

[PJ_SIM_SUNBNB_2023-12 [FE] Home Page Card Component](https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-12)

# Slack Link
https://sun-philippine.slack.com/archives/C05GURS7N3U/p1690462045152349

# Definition of Done
Create a card component for the listings containing:
- [x] Image
- [x] Label
- [x] Ratings
- [x] Reviews
- [x] Price
- [x] Type
- [x] Capacity

# Pre-conditions 
none

# Expected Output
image

# Screenshots
image

### CODE REVIEW

File: fe/src/components/HomeCard.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<template>
+    <v-app>
+      <v-main>
+        <h2>Homes</h2>
+        <v-container>
+          <v-layout>
+            <div v-for="home in homes" :key="home.id" class="pl-3 pr-11">
```
Comment (LennardUy104) [inline fe/src/components/HomeCard.vue]: Dont use for loop, just create one card.

please refer to the photo below

image

File: package-lock.json
Comment (LennardUy104) [inline package-lock.json:1]: What is this for?

**Review by LennardUy104 — CHANGES_REQUESTED** [review]
Please Create one card only. Thanks

File: fe/src/components/HomeCard.vue (hunk 1 above)
Comment (ThereseRoca) [reply fe/src/components/HomeCard.vue]: I have already resolved it. Thank you for your time and insights.

File: package-lock.json
Comment (ThereseRoca) [reply package-lock.json:1]: I think it was automatically included when I made my first commit but for my second commit it is not included anymore. Thank you.

**Review by LennardUy104 — APPROVED** [review]
lgtm! thanks

**Pull request closed without merging**